src/main/scala/tech/cryptonomic/conseil/routes/DataHelpers.scala

```diff
@@ -0,0 +1,90 @@
+package tech.cryptonomic.conseil.routes
+
+import java.sql.Timestamp
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Directives.complete
+import akka.http.scaladsl.server.Route
+import cats.Functor
+import endpoints.akkahttp
+import endpoints.algebra.Documentation
+import tech.cryptonomic.conseil.generic.chain.DataTypes.{AnyMap, QueryValidationError}
+import tech.cryptonomic.conseil.routes.openapi.{DataEndpoints, QueryStringListsServer, Validation}
+import tech.cryptonomic.conseil.tezos.Tables
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait DataHelpers extends QueryStringListsServer with Validation with akkahttp.server.Endpoints
+  with akkahttp.server.JsonSchemaEntities with DataEndpoints {
+
+  import io.circe._
+  import io.circe.syntax._
+
+  override def validated[A](response: A => Route, invalidDocs: Documentation): Either[List[QueryValidationError], A] => Route = {
+    case Left(errors) =>
+      complete(StatusCodes.BadRequest -> s"Errors: \n${errors.mkString("\n")}")
+    case Right(success) =>
+      response(success)
+  }
+
+  protected def optionFutureOps[M](x: Option[Future[M]])(implicit ec: ExecutionContext): Future[Option[M]] =
+    Future.sequence(Option.option2Iterable(x)).map(_.headOption)
+
+  protected def eitherOptionOps[A, B](x: Either[A, Option[B]]): Option[Either[A, B]] = x match {
+    case Left(value) => Some(Left(value))
+    case Right(Some(value)) => Some(Right(value))
+    case Right(None) => None
+  }
+
+  override implicit def qsFunctor: Functor[QueryString] = new Functor[QueryString] {
+    override def map[From, To](f: QueryString[From])(map: From => To): QueryString[To] = new QueryString[To](
+      f.directive.map(map)
+    )
+  }
+
+  def anyEncoder: Encoder[Any] = (a: Any) => a match {
+    case x: java.lang.String => Json.fromString(x)
+    case x: java.lang.Integer => Json.fromInt(x)
+    case x: java.sql.Timestamp => Json.fromBigInt(x.getTime)
+    case x: java.lang.Boolean => Json.fromBoolean(x)
+    case x: scala.collection.immutable.Vector[Tables.OperationGroupsRow] => x.map(_.asJson(operationGroupsRowSchema.encoder)).asJson
+    case x: Tables.BlocksRow => x.asJson(blocksRowSchema.encoder)
+    case x => Json.fromString(x.toString)
+  }
+
+  override implicit def anySchema: JsonSchema[Any] = new JsonSchema[Any] {
```

Comment:
Same considerations as above, a schema for `Any` type sounds like a runtime failure waiting to happen somewhere in the future.
Any alternative?

Reply:
Let's record here that we agreed to stick with this, and see if we can do better later on